src/client/datascience/jupyter/debuggerVariables.ts

```diff
@@ -236,6 +236,32 @@ export class DebuggerVariables extends DebugLocationTracker
         // When the initialize response comes back, indicate we have started.
         if (message.type === 'response' && message.command === 'initialize') {
             this.debuggingStarted = true;
+        } else if (
+            (message as DebugProtocol.StackTraceResponse).command === 'stackTrace' &&
```

Comment:
DebuggerVariables should be reacting to the `stopped` event, not assuming that a stackTrace request will go through at the right time.